networkx/algorithms/d_separation.py

```diff
@@ -11,6 +11,59 @@
 algorithm presented in [2]_.  Refer to [3]_, [4]_ for a couple of
 alternative algorithms.
 
+Here, we provide a brief overview of d-separation and related concepts that
+are relevant for understanding it:
+
+Blocking paths
+--------------
+
+Before we overview, we introduce the following terminology to describe paths:
+
+- "open" path: A path between two nodes that can be traversed
+- "blocked" path: A path between two nodes that cannot be traversed
+
+A **collider** is a triplet of nodes along a path that is like the following:
+``... u -> c <- v ...``), where 'c' is a common successor of 'u' and 'v'. A path
+through a collider is considered "blocked". When
+a node that is a collider, or a descendant of a collider is included in
+the d-separating set, then the path through that collider node is "open". If the
+path through the collider node is open, then we will call this node an open collider.
+
+If a path between 'u' and 'v' contains a non-collider node that is included in the
+d-separation set, then that path is "blocked". In other words, the d-separation set
```

Comment:
I think the single quotes in this entire section should be double-back-ticks e.g., ``u`` so that the font is changes for those names (they aren't input parameters because this isn't a function's doc_string. input parameters would get single back-ticks).